playbooks/archivematica-xenial/Vagrantfile

```diff
@@ -31,7 +31,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   # Make the project root available to the guest VM
-  config.vm.synced_folder '.', '/vagrant'
+  config.vm.synced_folder '.', '/home/vagrant-sync'
```

Comment:
IIRC `/vagrant` is a default synced folder configured by Vagrant and the original L34 wasn't really needed. I think that what this means is that with your change you would have a new `/home/vagrant-sync` but `/vagrant` would still be configured automatically. I haven't confirmed it but I have a strong feeling that this is the case.

If you wanted to disable `/vagrant` I think that you'd need to add the additional line:

    config.vm.synced_folder ".", "/vagrant", disabled: true

In any case, I'd still recommend to submit this as a separate issue/pull-request.